### PR TITLE
Wiki issue 63

### LINKED ISSUE #63: Document toolbar_bloc.dart
The overall purpose of documenting the program is to describe the overall structure of the program and not specifically code.

For solving this issue you can do the following:

Relations:
- Write about which BLoCs/Screens this class is related to.

Responsibility/Functionality:
- Write about the responsibility/functionality of the current class and how it interacts with it’s related BLoCs/Screens (e.g. this stream is used to send weekplans to this screen, so they can be displayed).

### PR DESCRIPTION
**Pull request by User12837985123**
This closes #63

### CODE REVIEW

**Comment by User12837985123** [comment]
@aau-giraf/sw608f19 you have been assigned this. Here's your checklist. I marked @swuf as the reviewer, but you might change it as you like.
Please use the following checklist:
- [ ] Is there documented which BLoCs/Screens the current class is related to?

- [ ] Is the responsibility/functionality of the current class explained? Also regarding the classes it is related to?

- [ ] If any unusual decisions are introduced in the code are they reasoned about in the documentation?

File: file_documentation/weekplanner/blocs/toolbar_bloc.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Documentation of toolbar_bloc.dart
+This file documents the `toolbar_bloc.dart` file from the weekplanner project.
+
+## Functionality
+`toolbar_bloc.dart` is responsible for emitting icons through the stream `visibleButtons` which is used by the Giraf appbar to determine which icons it should display.
```
Comment (ChristianBroS) [inline file_documentation/weekplanner/blocs/toolbar_bloc.md:5]: Isn't it also responsible for dialogs?

File: file_documentation/weekplanner/blocs/toolbar_bloc.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+## Code
```
Comment (ChristianBroS) [inline file_documentation/weekplanner/blocs/toolbar_bloc.md:7]: Nothing about the dialogs in this section either.

**Review by ChristianBroS — CHANGES_REQUESTED** [review]

File: file_documentation/weekplanner/blocs/toolbar_bloc.md (hunk 1 above)
Comment (User12837985123) [reply file_documentation/weekplanner/blocs/toolbar_bloc.md:5]: Yes, some of the functionality of the icon buttons are to show a dialog. I added a little at the end of the Code section.

**Review by ChristianBroS — APPROVED** [review]

**Pull request merged**